packages/components/select/src/select.vue

```diff
@@ -335,12 +335,29 @@ export default defineComponent({
   ],
 
   setup(props, { emit }) {
-    const API = useSelect(props, emit)
+    const modelValue = computed(() => {
+      const { modelValue: rawModelValue, multiple } = props
+      const fallback = multiple ? [] : undefined
```

Comment:
The fallback value of the original single choice should be an empty string. Should we keep it consistent with the original one?

Reply:
This is consistent with SelectV2, I feel it is more appropriate this way.

Reply:
Do we need to update the bound value as before?

![image](https://github.com/user-attachments/assets/5e4ef729-ed6f-45f5-a117-ca3ba289b5f5)

Reply:
This will create new issues, link #17279

Reply:
I see.